chaos/navitia.py

```diff
@@ -83,7 +83,13 @@ def _navitia_caller(self, query):
     def get_pt_object(self, uri, object_type, pt_objects=None):
         cache_key = 'Chaos.get_pt_object.{}.{}.{}.{}.{}'.format(self.coverage, self.get_coverage_publication_date(),
                                                                 uri, object_type, pt_objects)
-        cached_pt_object = cache.get(cache_key)
+        try:
+            cached_pt_object = cache.get(cache_key)
+        except:
+            logging.getLogger(__name__).error('Redis time out !')
```

Comment:
```suggestion
            logging.getLogger(__name__).error('Cache Timeout')
            logging.getLogger(__name__).info('Trying to call Navitia')
```
(pour plus générique si demain on ne passe plus par Redis)